### ISSUE TITLE
Installation is very hard on Windows

### ISSUE THREAD

**Issue by tngreene** [python-rope/ropevim#68]
Hi,

I was very excited to try using this in VIm, but found it to be a frustrating mess. I'm filing some bugs for each problem I encountered.

Details:
I'm using gVim 8, huge features, compiled Sept 6th, 2017.
- Windows 10 x64 bit
- Python 3 installed
- ropevim and ropemode were installed using 
```viml
Plug 'python-rope/ropemode'
Plug 'python-rope/ropevim'
```

Firstly, installation:
1. The instructions said "New Features: ropemode is not the part of distribution now", which I understood as "This is a good thing!" Not, "this is secretly a requirement and you'll only find out that when you dig into the setup.py for ropevim or notice and understand ``ImportError: No module named ropemode.decorators`` "

2. "Add ropevim folder to PYTHONPATH" is unclear. After running the setup script, a compiled copy of rope_omni and ropevim were put in "C:\Vim\Lib\site-packages" (yes, I know that is an odd place for it to go). Adding "C:\Vim\Lib\site-packages" made it work. So

- Is this my path issue, and all compiled ropevim files should have stayed in my bundle folder?
- Is this a poor piece of instruction?
- Is there supposed to be a copy in .vim/bundle and in this other place, and both are supposed to be in the PYTHONPATH?

3. The install script was not very clean. Running it did not put the contents in their own folder, ``ropevim``, like it's brother ``ropemode``. This was a bit annoying to deal with. and made me unsure if I was using it right/putting it in the right place. This is my folder at the end of the day.

```
rope
rope-0.10.7.dist-info
ropemode
ropemode-0.3-py3.6.egg-info
ropevim-0.7.0-py3.6.egg-info
ropevim.py
ropevim.pyc
rope_omni.py
rope_omni.pyc
```

I am somewhat new at python's package managment system and don't know what the norms are. An issue I had was that the only way to get my copy of Vim to work with python was to copy a whole Python 3.6 installation into the Vim folder, which is probably why C:\Vim\Lib\site-packages was where it thought these things should go, right?

4. I don't know if there is a way to increase the error messages on the script, but having it say "You don't have ropemode/you don't have it in the right place!" would have been a big help. Otherise it said that it had completed. Which is true, just that its not the same as it working.

Making the instructions a little clearer would have been a big help. I did get it loading, but not really working.

**Comment by mcepl** [comment]
Yeah, it is not very clearly said. “Not part of distribution” means in fact “you should install it via ``pip install ropevim``”. Then all dependencies install as well. Pull request explaining installation for normal humans in ``README`` would be gratefully accepted.
